docs/resources/bridge_domain.md

```diff
@@ -227,6 +227,10 @@ All examples for the Bridge Domain resource can be found in the [examples](https
 * `name_alias` (nameAlias) - (string) The name alias of the Bridge Domain object.
 * `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
 * `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `pc_tag` (pcTag) - (string) The classification tag used for policy enforcement and zoning.
+* `scope` (scope) - (string) The L3 scope ID of the Bridge Domain object.
+  - Default: `0`
```

Comment:
Should we specify a Default for this, if it is a Read Only property?

Reply:
Yes but it should also not be under optional. We already discussed this issue. Created a new issue to track this: https://github.com/CiscoDevNet/terraform-provider-aci/issues/1303